lib/manager/components/LabelPanel.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React, { Component } from 'react'
+import { Panel } from 'react-bootstrap'
+
+import FeedLabel from '../../common/containers/FeedLabel'
+import type { ManagerUserState } from '../../types/reducers'
+
+import LabelEditorModal from './LabelEditorModal'
+
+export type Props = {
+  large?: boolean,
+  project?: any,
+  user: ManagerUserState,
+};
+
+export default class LabelPanel extends Component<Props> {
+  _onClickNewLabel = () => this.refs.newLabelModal.open();
+
+  render () {
+    const { large, project, user } = this.props
+
+    // make sure there is a valid
+    // project we can add labels tobefore continuing
+    if (!project || !project.labels || !project.id) {
+      return <div />
```

Comment:
It might be clearer if null was returned instead of an empty div.

Reply:
I've changed this in a1bc3e9, but doesn't this sometimes cause white screens?